.circleci/config.yml

```diff
@@ -199,6 +199,18 @@ jobs:
           path: ~/e2e/test-results
       - store_artifacts:
           path: ~/e2e/artifacts
+  grace_nightly:
+    machine: true
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - run: docker login -u=$QUAY_USER -p=$QUAY_PASS quay.io
+      - run: docker run --net host -v /var/run/docker.sock:/var/run/docker.sock -v ~/project:/grace/grace-results quay.io/influxdb/grace:latest
+      - store_artifacts:
+          path: ~/project
+          destination: grace-nightly-output
+      - store_test_results:
+          path: ~/project/grace-results
```

Comment:
@stephanie-engel is it supposed to be `path: ~/project` and not `path: ~/project/grace-results`?

Reply:
Thanks for catching that! I must've accidentally re-added it. I just removed it!